scripts/supercloud/run_option_learning_experiments.sh

```diff
@@ -20,26 +20,34 @@ for SEED in $(seq $START_SEED $((NUM_SEEDS+START_SEED-1))); do
         # NOTE: segmenter is oracle
         COMMON_ARGS="--env stick_point --min_perc_data_for_nsrt 1 \
             --segmenter oracle --num_train_tasks $NUM_TRAIN_TASKS --timeout 300 \
-            --seed $SEED"
+            --seed $SEED --gnn_num_epochs 10000"
```

Comment:
5k epochs took about 3.25 hours, so we can afford to double it

Reply:
clarifying for future, the PR only shows adding a new flag here, which we are doing because the default of 25K is too high when doing gnn_action_policy with multistep options (there is a lot more data than with gnn_option_policy)